src/modules/real-time-metrics/reports/convert-beholder-to-chart.js

```diff
@@ -331,15 +331,10 @@ const formatTsChartData = ({
 
   const seriesArray = fillSeriesWithZeroes(series, countValues)
 
-  if (report.id === '071851224118431167') {
+  const BOT_CAPTCHA_CHART = '071851224118431167'
+  if (report.id === BOT_CAPTCHA_CHART) {
     seriesArray.forEach((serie) => {
-      // linter complains about ternary
-      if (serie[0] === 'false') {
-        serie[0] = 'Not Solved'
-      }
-      if (serie[0] === 'true') {
-        serie[0] = 'Solved'
-      }
+      serie[0] = serie[0] === 'true' ? 'Solved' : 'Not Solved'
```

Comment:
Maybe for better readability we could go for something like this
```suggestion
      serie[0] = (serie[0] === 'true') ? 'Solved' : 'Not Solved'
```

Reply:
The formatter ends up removing it.